script/grblcon.py

```diff
@@ -656,7 +781,9 @@ def _open_port(self, portstr='/dev/ttyUSB0', baud=38400):
                 self._end_evt.clear()
                 self._serial_thread.start()
             except (serial.serialutil.SerialException, FileNotFoundError):
-                self._log_crit("Unable to open port: {}".format(portstr))
+                self._log_error(
+                    "Unable to open port: {}".format(portstr), True
+                )
 
     def poll_adcs(self, *args):  # pylint: disable=unused-argument
         """Periodically polls the ADC values in grbl
```

Comment:
The comment is not for the line, but for the file. Sorry, the comment is in the wrong place.